src/options.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const DEFAULT_PING_INTERVAL = 30_000
+const DEFAULT_MAX_RECONNECTION_RETRIES = Infinity
```

Comment:
Max retries should be limited by default.
There is no reason to reconnected unlimited since in reconnecting period the client may continue sending data.

Reply:
The intention is to keep the connection alive, assuming the error messages are being monitored
We can remove the default value, but in that case we can't set an arbitrary default value

Reply:
I agree. Those should be limited, possibly to 30 seconds.

Reply:
Ok I'll work on it, I'll add a buffer on incoming messages